src/github-repo-event-notifier.coffee

```diff
@@ -102,8 +103,8 @@ module.exports = (robot) ->
 
     res.end ""
 
-announceRepoEvent = (data, eventType, cb) ->
+announceRepoEvent = (adapter, data, eventType, cb) ->
   if eventActions[eventType]?
-    eventActions[eventType](data, cb)
+    eventActions[eventType](data, cb, adapter)
```

Comment:
I'm OK with this though it's usually customary (from what I remember) for the callback (`cb` here) to be the last parameter. That is kind of annoying to go back and change so if you don't want to do it, no sweat. 😄